package.json

```diff
@@ -55,36 +55,36 @@
     "docs:serve": "vitepress serve docs"
   },
   "peerDependencies": {
-    "lexical": "0.15.0",
+    "lexical": "^0.16.0",
     "vue": "^3.2.0"
   },
   "dependencies": {
-    "@lexical/clipboard": "0.15.0",
-    "@lexical/code": "0.15.0",
-    "@lexical/dragon": "0.15.0",
-    "@lexical/hashtag": "0.15.0",
-    "@lexical/history": "0.15.0",
-    "@lexical/html": "0.15.0",
-    "@lexical/link": "0.15.0",
-    "@lexical/list": "0.15.0",
-    "@lexical/mark": "0.15.0",
-    "@lexical/markdown": "0.15.0",
-    "@lexical/overflow": "0.15.0",
-    "@lexical/plain-text": "0.15.0",
-    "@lexical/rich-text": "0.15.0",
-    "@lexical/selection": "0.15.0",
-    "@lexical/table": "0.15.0",
-    "@lexical/text": "0.15.0",
-    "@lexical/utils": "0.15.0",
-    "@lexical/yjs": "0.15.0",
+    "@lexical/clipboard": "^0.16.0",
```

Comment:
@wobsoriano what are the pros and cons with exactly pinning the version? So far it has been easier for us to work with ones that are pinned exactly

Reply:
Using the `^x.x.x` method can be effective as it eliminates the need to manually update lexical-vue with every patch version change. However, if you choose to pin the version exactly, you'll need to manually update it whenever there is a new release of lexical, but this approach does make it clear which version you are using and simplifies management. Additionally, implementing Dependabot could be an efficient way to manage updates.

Reply:
@taishinaritomi SLR, but using a caret will allow any version less than 1.0.0 right? What if we use tilde (`~`) instead? So that the version should at least be 0.16.0, but not 0.17.0? Thanks!

Reply:
and maybe we can just move the `lexical` package to `dependencies`

Reply:
Also, I added changesets for automatic releases. Would you be able to rebase and generate a changeset via `npx changeset add`? The bot didn't pick up this PR since it's an old one

Reply:
Ill do it on my end. Thanks!